en/docs/integrations/integration-concepts.md

```diff
@@ -1,57 +1,54 @@
- - **Google Sheets**: To interact with the Google Sheets API and perform operations such as creating and listing spreadsheets.
- - **Salesforce**: To interact with the Salesforce API and perform required operations. 
- - **NetSuite**: To interact with the NetSuite API and perform required operations. 
+ - **Twilio**: Connects your service to Twilio API to perform operations such as sending messages, WhatsApp messages, and making voice calls.
+ - **Slack**: Connects your service to Slack API to perform operations in Slack, such as sending a message to a channel, deleting a message, etc.
+ - **GitHub**: Connects your service to GitHub API to perform operations in GitHub, such as creating an issue, creating a pull request, etc. 
+ - **Gmail**: Connects your service to Gmail API to perform email operations such as sending an email, receiving an email, etc.
+ - **Google Calendar**: Connects your service to Google Calendar API to perform operations in Google Calendar such as creating an event, deleting an event, etc.
+ - **Google Sheets**: Connects your service to Google Sheets API to perform operations in Google Sheets such as reading data, formatting data, etc.
+ - **Salesforce**: Connects your service to Salesforce API to perform Salesforce operations such as obtaining organization information, creating a record, etc.
+ - **NetSuite**: Connects your service to Netsuite API to perform Netsuite operations such as creating a record instance, creating a sublist, etc. 
 
 ## Statement
-A statement in Choreo integrations is a syntactic unit that specifies the action to be carried out in an integration. 
 
-Choreo supports the following statements:
+A statement is a collection of code fragments that specifies the actions a Choreo integration will carry out. Choreo statements can also be composed of expressions. These expressions are written based on the [Ballerina Expression Syntax](#ballerina-expression-syntax).
 
- - **Log**: To add an entry to the log at runtime. You can either add an **Info** log or an **Error** log depending on your requirement.   
- - **Variable**: To declare a variable of a preferred type depending on your requirement.
- - **If**: To perform a conditional check and evaluate whether a statement is true or false.   
- - **ForEach**: To iterate over a list of items to perform control flow. 
- - **While**: To iterate over a given condition to perform control flow. 
- - **Return**: To end the ongoing execution of an integration and return control to the caller.
- - **Custom**: To specify a custom statement depending on your requirement.
+Choreo supports the following statements:
 
-## Expression Syntax
-The syntax to use when you want to input values for an expression when designing an integration using Choreo. [Refer to the syntax.](../references/choreo-expression-syntax)
+ - **Log**: Logs an event with an information statement or an error that occurs in your service.    
```

Comment:
Why do we say service here instead of integration? Check all the places.

Reply:
Isn't it that you are actually connecting your service to an external system in order to create an integration?

Reply:
No. We don't needt to have a service to create an `integration` in choreo.

Reply:
Fixed.